src/app/components/library/library.rs

```diff
@@ -131,6 +134,13 @@ impl EventListener for Library {
             AppEvent::LoginEvent(LoginEvent::LoginCompleted(_)) => {
                 let _ = self.model.refresh_saved_albums();
             }
+            AppEvent::BrowserEvent(BrowserEvent::LibraryUpdated) => {
+                if let Some(list) = self.model.get_list_store() {
```

Comment:
(minor) you could add a has_albums method to the model

Reply:
depends how you see it I guess, but what you care about locally is whether there are albums or not (and this avoids having to deal with the option here) but im fine with either :)

Reply:
That's true. I overlooked it and refreshing the albums is done elsewhere so a `has_albums` method is actually the better for this case :)